cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -249,15 +249,15 @@ def validate_and_update_from_json(cls, descriptor, jsondict, user, filter_tabs=T
                 errors.append({'message': text_type(err), 'model': model})
 
         # Disallow updates to the proctoring provider after course start
-        proctoring_provider_model = filtered_dict.get('proctoring_provider')
+        proctoring_provider_model = filtered_dict.get('proctoring_provider', {})
 
         # If the user is not edX staff, the user has requested a change to the proctoring_provider
         # Advanced Setting, and and it is after course start, prevent the user from changing the
         # proctoring provider.
         if (
             not user.is_staff and
             cls._has_requested_proctoring_provider_changed(
-                descriptor.proctoring_provider, proctoring_provider_model
+                descriptor.proctoring_provider, proctoring_provider_model.get('value', None)
```

Comment:
Style nit: `None` can be omitted because it's already the default fallback value for `.get(key)`

Reply:
Ah! I didn't know that. Thank you.